pushing_testing_files/package.json

```diff
@@ -7,10 +7,11 @@
     "test": "tests"
   },
   "scripts": {
-    "langs": "npm run build_langs && npm run cucumber",
-    "build_langs": "node_modules/.bin/da-langs 'tests/features/*.feature'",
+    "test": "npm run langs_setup && npm run cucumber",
     "cucumber": "./node_modules/.bin/cucumber-js --require ./node_modules/docassemble-cucumber/lib/index.js tests/features/*.feature",
-    "local_langs": "npm run build_langs && npm run local",
+    "langs": "npm run langs_setup && ./node_modules/.bin/cucumber-js --require ./node_modules/docassemble-cucumber/lib/index.js tests/features/*__lang*.feature",
```

Comment:
should this be something like `langs_test` to mirror `test`?

Reply:
alternatively, it would be nice to have a description of when one would use each of these commands. not a blocker for this MR, but a nice-to-have piece of documentation.

Reply:
I'm not sure how to add a description as I've never done that in JSON (which doesn't have comments). `test` is for github, which runs the workflow, which uses `setup` and `takedown` in their own steps. `local` runs `setup` and `takedown` right in the script itself. The `local_langs`/`langs_local` script uses `local` inside it.

I'd appreciate your thoughts on it, though. I honestly find myself only using `npm run local` very rarely, though it is nice to have when I do use it.

Reply:
I'll pull for now and maybe we can talk about this choice later.

Reply:
I was thinking in the readme file, we could have a description of all the default commands the package has. I'm not wedded to that idea though, so if you have a better location for this documentation, lmk.